common/test/acceptance/pages/studio/video/video.py

```diff
@@ -55,6 +55,7 @@
     # basic
     [DISPLAY_NAME, 'Video', False],
     ['Default Video URL', 'https://www.youtube.com/watch?v=3_yD_cEKoCk, , ', False],
+    ['Video ID', '', False],
```

Comment:
Why are we setting an empty string here, don't we need some meaning full text here ?